guides/keras-tuner/subclass_tuner.py

```diff
@@ -0,0 +1,193 @@
+"""
+Title: Subclassing Tuner for Custom Training Loops
```

Comment:
We only capitalize class names in titles (see https://keras.io/guides/)

In this case please switch the title to "Writing your own Tuner to support a custom training loop"